_content/word-list.md

```diff
@@ -217,137 +262,170 @@ internet
 ### J
 
 join a channel
-: Use instead of *enter a channel*.
+: Instead of *enter a channel*.
 
 just
-: Avoid. Usually, *just* is a filler word that you can delete without affecting your meaning.
+: Avoid. Typically a filler word that can be deleted.
 
 ### L
 
 last mile
-: Use as a noun or adjective. Always two separate words.
+: As a noun or adjective.
 
 later
-: Use instead of *higher* when describing a version.
+: Instead of *higher* when describing a version.
+
+listen for
+: Instead of *listen to* when referring to callback events.
 
 live stream
-: Use as an intransitive verb or noun.
+: As an intransitive verb or noun.
 
 live streaming
-: Use *live streaming* to refer to Agora products and scenarios.
+: Refers to Agora products and scenarios.
 
 log in
-: Use as a verb.
+: As a verb.
 
 login
-: Use as a noun or adjective.
+: As a noun or adjective.
 
 ### M
 
+maximum
+: Instead of *at most*.
+
 might
-: Use instead of may when referring to possibility.
+: Instead of *may* when referring to a possibility.
 
 multichannel
-: No hyphen.
+: &nbsp;
 
 multimedia
-: No hyphen.
+: &nbsp;
 
 multithreaded
-: No hyphen.
+: &nbsp;
 
 ### N
 
 number
-: Use instead of *no.*
+: Instead of *no.*
 
 ### O
 
 open source
-: No hyphen, not even as an adjective or verb.
+: As an adjective or verb.
+
+or not
+: Avoid. Unnecessary in phrases such as *whether or not*.
 
 ### P
 
 packet loss rate
 : &nbsp;
 
 page
-: Use *this page* instead of *this article*.
+: Instead of *article* when referring to documentation.
+
+pass
+: Instead of *pass in* when referring to an object as an argument.
 
 path
-: Avoid using *filepath*, *file path*, *pathname*, or *path name* if possible.
+: Avoid using *filepath*/*file path* and *pathname*/*path name*, if possible.
 
 plugin
-: Use as a noun.
+: As a noun.
 
 plug-in
-: Use as an adjective.
+: As an adjective.
 
 plug in
-: Use as a verb.
+: As a verb.
 
 ### R
 
 real time
-: Use as a noun or adverb.
+: As a noun or adverb.
 
 real-time
-: Use as an adjective.
+: As an adjective.
+
+reduce
+: Instead of *save* in phrases such as *reduce bandwidth*.
 
 reintegrate
 : No hyphen.
 
+reserved parameter
+: Instead of *reserved for future use*.
+
 ### S
 
+sample code
+: Instead of *sample codes*. //TODO William Is this an issue? I think "sample codes" is just incorrect?
+
 screen sharing
-: Use as a noun or adjective.
+: As a noun or adjective. //TODO William Are we sure? "screen-sharing stream" and "screen-sharing scenario" appear in the Dev Center? 
```

Comment:
I googled "screen-sharing" and did not see many results. Maybe we should update existing docs?